app/data.py

```diff
@@ -128,6 +128,24 @@ def read(self, collection: str, query: Optional[Dict] = None) -> List[Dict]:
         """
         return list(self.get_collection(collection).find(query, {"_id": False}))
 
+    def projection(self, collection: str, query: Dict, projection: Dict) -> List[Dict]:
```

Comment:
Consider renaming this function just for clarity purposes since future graphs might want to implement different projections, maybe change to "feedback_projection" (idk if I misunderstood Robert's API graph tutorial though so name might be appropriate and it's my mistake but reconsider)

Reply:
The projection method is not just for feedback. It's more general. This name is fine.

Reply:
Ok, thanks for clarifying Robert, I just figured future cohorts would be using projections with different fields.